content/md/en/docs/tutorials/collectibles-workshop/03-create-pallet.md

```diff
@@ -3,6 +3,8 @@ title: Create a new pallet
 tutorial: 1
 ---
 
+> ⚠️ This tutorial is out-of-date any may not work as intended. Please refer to [`Polkadot SDK Docs`](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/guides/your_first_pallet/index.html) for up-to-date information.
```

Comment:
@kianenigma is there something more fitting to link here or is this the intended follow-up?